ontology.ttl

```diff
@@ -8,7 +8,7 @@
 @prefix foaf: <http://xmlns.com/foaf/0.1/> .
 @prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
 @prefix vann: <http://purl.org/vocab/vann/> .
-@base <http://odahub.io/ontology> .
+@base <http://odahub.io/ontology#> .
```

Comment:
why this?

Reply:
It was defined initially like this, then changed unintentionally (consequence of Protégé preferences).
But anyway, we don't use this base anywhere in the ttl file, relying instead on empty prefix.

Reply:
It would be better to stick to some fixed form so we do not introduce changes with little or no implications for the ontology. This is similar to style changes which should be handled separately.
But ok, let's hope this fixes some style for now.